platforms/blabsy/src/components/tweet/tweet.tsx

```
@@ -20,187 +20,213 @@ import type { Tweet } from '@lib/types/tweet';
+                        <AnimatePresence initial={false}>
+                            {modal ? null : pinned ? (
+                                <TweetStatus type='pin'>
+                                    <p className='text-sm font-bold'>
+                                        Pinned Blab
+                                    </p>
+                                </TweetStatus>
+                            ) : (
+                                tweetIsRetweeted && (
+                                    <TweetStatus type='tweet'>
+                                        <Link href={profileUsername as string}>
+                                            <a className='custom-underline truncate text-sm font-bold'>
```

Biome (1.9.4)

[error] Provide a href attribute for the a element.

An anchor element should always have a href
Check this thorough explanation to better understand the context.

(lint/a11y/useValidAnchor)